lib/tophat.rb

```diff
@@ -21,13 +21,14 @@ def reset
   end
 
   def setup
-    ActionView::Base.send :include, TopHat::HtmlHelper
-    ActionView::Base.send :include, TopHat::TitleHelper
-    ActionView::Base.send :include, TopHat::MetaHelper
-    ActionView::Base.send :include, TopHat::StylesheetHelper
-    ActionView::Base.send :include, TopHat::RobotsHelper
-    ActionView::Base.send :include, TopHat::OpenGraphHelper
-    ActionView::Base.send :include, TopHat::TwitterCardHelper
+    ActiveSupport.on_load do
```

Comment:
Also, not all projects are subject of the issue being discussed at https://github.com/rails/rails-controller-testing/issues/24